### PR TITLE
update version number and authors 

### PR DESCRIPTION
**Pull request by imyiningliu**

### CODE REVIEW

File: HISTORY.rst [hunk 1]
```diff
@@ -3,6 +3,8 @@ History
 =======
 
 0.1.0 (2018-03-28)
+0.2.0 (2018-06-12)
 ------------------
 
 * First release on PyPI.
+* Second release on PyPI.
```
Comment (jeff-regier) [inline HISTORY.rst]: It's our first release on PyPI.

File: scvi/__init__.py [hunk 2]
```diff
@@ -2,6 +2,10 @@
...
 
-__author__ = """Romain Lopez"""
-__email__ = 'romain_lopez@berkeley.edu'
-__version__ = '0.0.1'
+__author__ = "Romain Lopez, Jeffrey Regier, Maxime Langevin, Edouard Mehlman, Yining Liu"
+__email__ = """Romain Lopez <romain_lopez@berkeley.edu>, 
+            Jeffrey Regier <jregier@eecs.berkeley.edu>, 
+            Maxime Langevin <maxime.langevin@polytechnique.edu>, 
+            Edouard Mehlman <edouard.mehlman@polytechnique.edu>, 
+            Yining Liu <lyining@berkeley.edu>"""
```
Comment (jeff-regier) [inline scvi/__init__.py]: Do other projects list a lot of authors? If it's more normal to list just one author, I'd just list Romain.

File: HISTORY.rst (hunk 1 above)
Comment (imyiningliu) [reply HISTORY.rst]: Should I remove the two lines about 0.1.0?

File: scvi/__init__.py (hunk 2 above)
Comment (imyiningliu) [reply scvi/__init__.py]: It's more common to have one author. I'll change it back.

File: HISTORY.rst (hunk 1 above)
Comment (jeff-regier) [reply HISTORY.rst]: Oh, I'm not sure. Maybe we should just call this release 0.1.0 after all.

Comment (jeff-regier) [reply HISTORY.rst]: All the text that is there now is from when I started the project by using template-generating code called cookiecutter: https://github.com/audreyr/cookiecutter

So no need to take the files generated by cookiecutter too seriously. They're not particularly customized to our project.

File: setup.py [hunk 3]
```diff
@@ -4,37 +4,53 @@
...
 with open('README.rst') as readme_file:
     readme = readme_file.read()
 
 with open('HISTORY.rst') as history_file:
     history = history_file.read()
 
-requirements = []
+
+req_path = Path('requirements.txt')
+if not req_path.is_file():
```
Comment (jeff-regier) [inline setup.py]: We don't need this case -- it'll always be a file

**Comment by jeff-regier** [comment]
Looks good, please merge it whenever you're ready.

**Comment by jeff-regier** [comment]
By selecting "Squash and merge" from the green drop-down menu.

**Pull request merged**